client/src/components/Home/index.js

```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
               <p>
                 <a href="#" className="header-link">
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
+                </a>{"https://docs.rocket.chat/guides/user-guides"}
```

Comment:
The `{"https://docs.rocket.chat/guides/user-guides"}` after the `a` tag is currently rendering as a [text](https://i.postimg.cc/zBShRFYc/Screenshot-from-2021-03-24-19-50-42.png). We need to remove this.

Reply:
ok